crates/escrow/src/lib.rs

```diff
@@ -82,6 +82,16 @@ impl<Balance: AtLeast32BitUnsigned + Copy, BlockNumber: AtLeast32BitUnsigned + C
         let height_diff = BlockNumberToBalance::convert(height.saturating_sub(self.ts));
         self.bias.saturating_sub(self.slope.saturating_mul(height_diff))
     }
+
+    // temporary function to limit stakeholder deposits
+    fn reverse_balance_at<BlockNumberToBalance: Convert<BlockNumber, Balance>>(
```

Comment:
Can you please elaborate more on what this function does? I have no idea what a "reverse balance" is supposed to mean.

Also, you are ignoring `self.ts`, is that on purpose?

Reply:
Updated the comments, the purpose of this function is to serve as the reverse of the normal `balance_at` function - i.e. linearly increase the amount.